pkg/model/replication.go

```diff
@@ -18,6 +18,24 @@ This module implements the common data structure.
 
 package model
 
+import "github.com/opensds/opensds/pkg/dock/proto"
+
+//volume status
+const (
+	ReplicationDeleted       = "deleted"
+	ReplicationCreating      = "creating"
+	ReplicationDeleting      = "enabling"
+	ReplicationEnabling      = "enabling"
+	ReplicationDisabling     = "disabling"
+	ReplicationFailovering   = "failovering"
+	ReplicationAvailable     = "available"
+	ReplicationError         = "error"
+	ReplicationErrorDeleting = "error_deleting"
+	ReplicationErrorEnabled  = "error_enabled"
```

Comment:
error_enabling or error_enable

Reply:
There should be a "failed_over" state as well.  Please take a look of this: https://github.com/openstack/cinder/blob/master/cinder/objects/fields.py#L100